modules/nw-metallb-configure-return-traffic-proc.adoc

```diff
@@ -0,0 +1,187 @@
+        - ip-to: 172.30.0.0/16
+          priority: 998
+          route-table: 254
+        - ip-to: 10.132.0.0/14
+          priority: 998
+          route-table: 254
+----
+<1> Name of the policy.
+<2> This example applies the policy to all nodes with the label `vrf:true`.
+<3> Name of the interface.
+<4> The type of interface. This example creates a VRF instance.
+<5> The node interface to which the VRF attaches.
+<6> Name of the route table ID for the VRF.
+<7> Defines the configuration for network routes. `next-hop-address` defines the IP address of the next hop for the route. `next-hop-interface` defines the outgoing interface for the route. In this example, the VRF's routing table is `2`, which references the ID that you define in the `EgressService` CR.
+<8> Defines additional route rules. The `ip-to`` fields must match the cluster IP address and pods CIDR in the networks.operator.openshift.io CR named cluster.
+//Does `oc describe network.config/cluster` get this information?
+
+.. Apply the configuration for the BGP peer by running the follow command:
++
+[source,terminal]
+----
+$ oc apply -f node-network-vrf.yaml
+----
+
+. Create a `BGPPeer` custom resources (CR).
+
+.. Create a file, such as `frr-via-vrf.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta2
+kind: BGPPeer
+metadata:
+  name: frrviavrf
+  namespace: metallb-system
+spec:
+  myASN: 100
+  peerASN: 200
+  peerAddress: 192.168.130.1
+  vrf: ens4vrf <1>
+----
+<1> Specifies the VRF instance to associate with the BGP peer. MetalLB can advertise services and make routing decisions based on the routing information in the VRF. 
+
+.. Apply the configuration for the BGP peer by running the follow command:
++
+[source,terminal]
+----
+$ oc apply -f frr-via-vrf.yaml
+----
+
+. Create an `IPAddressPool` CR.
+
+.. Create a file, such as `first-pool.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: first-pool
+  namespace: metallb-system
+spec:
+  addresses:
+  - 192.169.10.0/32
+----
+
+.. Apply the configuration for the IP address pool by running the follow command:
++
+[source,terminal]
+----
+$ oc apply -f first-pool.yaml
+----
+
+. Create a `BGPAdvertisement` CR.
+
+.. Create a file, such as `first-adv.yaml`, with content like the following example:
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: BGPAdvertisement
+metadata:
+  name: first-adv
+  namespace: metallb-system
+spec:
+  ipAddressPools:
+    - first-pool
+  peers:
+    - frrviavrf <1>
+  nodeSelectors: 
+    - matchLabels:
+        egress-service.k8s.ovn.org/test-server1: "" <2>
+----
+<1> In this example, MetalLB advertises a range of IP addresses from the `first-pool` IP address pool to the `frrviavrf` BGP peer.
+<2> To use the same return path for the traffic originating from the pod, select the node that handles the `LoadBalancer` service. 
```

Comment:
This is required only if the sourceIPBy is loadbalancer

Reply:
Adding "Optional" prefix to these callouts.